source/systems/gaea_user_guide.rst

```diff
@@ -91,2113 +93,1545 @@ points at Atlanta and Chicago.
+
+
+.. _gaea-login-nodes:
+
+Login nodes
+===========
+
+The Gaea login nodes have a similar architecture to the compute nodes.  Each
+compute cluster, C5 and C6, have a dedicated set of login nodes.
+
++----------------------+----------------------------+--------------------+
+| Host Names           | Node Configuration         | Associated Compute |
+|                      |                            | Cluster            |
++======================+============================+====================+
+| :regexp:`gaea5[1-8]` | 2x AMD EPYC 7662 64-core   | C5                 |
+|                      | (128 cores per node) with  |                    |
+|                      | 512 GiB of memory per node |                    |
++----------------------+----------------------------+--------------------+
+| :regexp:`gaea6[1-8]` | 2x AMD EPYC 9654 96-core   | C6                 |
+|                      | (192 cores per node) with  |                    |
+|                      | 512 GiB of memory per node |                    |
++----------------------+----------------------------+--------------------+
+
+.. _gaea-dtn-nodes:
+
+Data transfer nodes
+===================
+
+The data transfer nodes (DTN) is where extensive I/O operations, large local,
+and all off-gaea transfers should be done.  The :abbr:`DTN (Data Transfer
+Nodes)`\ s are accessible on the :dfn:`es` cluster, under the :dfn:`dtn_f5_f6`
+partition.
+
+The DTNs are the only systems that have both the :dfn:`f5` and :dfn:`f6`
+mounted.
+
++----------------------+----------------------------+--------------------+
+| Host Names           | Node Configuration         | File Systems       |
+|                      |                            | Mounted            |
++======================+============================+====================+
+| :regexp:`dtn[50-79]` | AMD EPYC 7302 16-core with || /gpfs/f5          |
+|                      | 256 GiB of memory per node || /gpfs/f6          |
++----------------------+----------------------------+--------------------+
+
+System interconnect
+===================
+
+The C5 and C6 nodes are connected with the HPE Slingshot.
+
++---------+--------------------------------------+-------------+
+| Cluster | :abbr:`NIC (Network Interface Card)` | Total       |
+|         |                                      | Bandwidth   |
++=========+======================================+=============+
+| C5      | [2x] HPE Slingshot 100 Gbps (12.5    | 200 Gbps    |
+|         | GB/s)                                |             |
++---------+--------------------------------------+-------------+
+| C6      | [1x] HPE Slingshot 200 Gbps (25.0    | 200 Gbps    |
+|         | GB/s)                                |             |
++---------+--------------------------------------+-------------+
 
-  c# refers to a computer cluster.
+File systems
+============
 
-Running
--------
+Each Gaea compute cluster, C5 and C6, have their own file system.  C5 has
```

Comment:
Recommend

Gaea compute clusters C5 and C6 have their own file systems.

Reply:
Update in a new push.